utils/organizations.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * There is only one metrics API endpoint to get these 3 values.
+ * The rest of the metrics aren't stored yet at the organization level
+ */
+export async function getOrganizationMetrics(oid: string) {
+  // Fetching last 12 months
+  const response = await fetch(`https://metric-api.data.gouv.fr/api/organizations/data/?organization_id__exact=${oid}&metric_month__sort=desc&page_size=12`)
+  const page = await response.json()
+
+  const datasetsViews: Record<string, number> = {}
+  const downloads: Record<string, number> = {}
+  const reusesViews: Record<string, number> = {}
+
+  for (const { metric_month, monthly_download_resource, monthly_visit_dataset, monthly_visit_reuse } of page.data) {
+    datasetsViews[metric_month] = monthly_visit_dataset
+    downloads[metric_month] = monthly_download_resource
+    reusesViews[metric_month] = monthly_visit_reuse
+  }
+  // Fetching totals
+  const totalResponse = await fetch(`https://metric-api.data.gouv.fr/api/organizations_total/data/?organization_id__exact=${oid}`)
```

Comment:
I don't know what's best but currently the total value displayed is only the sum of the year, not the total.